client/webpack.config.js

```diff
@@ -379,26 +363,10 @@ const webpackConfig = {
 				res.request = 'calypso/components/empty-component';
 			}
 		} ),
-		/*
-		 * Use "evergreen" polyfill config, rather than fallback.
-		 */
-		browserslistEnv === 'evergreen' &&
-			new webpack.NormalModuleReplacementPlugin(
-				/^@automattic\/calypso-polyfills$/,
-				'@automattic/calypso-polyfills/browser-evergreen'
-			),
-		/*
-		 * Local storage used to throw errors in Safari private mode, but that's no longer the case in Safari >=11.
-		 */
-		...( browserslistEnv === 'evergreen'
-			? [
-					new webpack.NormalModuleReplacementPlugin(
-						/^calypso[/\\]lib[/\\]local-storage-polyfill$/,
-						'lodash-es/noop'
-					),
-			  ]
-			: [] ),
-
+		new webpack.NormalModuleReplacementPlugin(
```

Comment:
This can be also changed in the source code, on each file using `@automattic/calypso-polyfills` (or even in the package). However this PR is already too big, I rather do that in a follow up PR.